### ISSUE TITLE
androidx.mediarouter:mediarouter:1.6.0-beta01 requires compileSdkVersion 34

### ISSUE THREAD

**Issue by KestasVenslauskas** [react-native-google-cast/react-native-google-cast#511]
**Describe the bug**
When I try to build android I get this error:

```
  1.  Dependency 'androidx.mediarouter:mediarouter:1.6.0-beta01' requires libraries and applications that
      depend on it to compile against version 34 or later of the
      Android APIs.
```

**To Reproduce**
Build on android device

**Expected behavior**
Build passes

**Device:**
 - Pixel 6a
 - Android 14
 - Library Version 4.6.2

**Comment by tobbbe** [comment]
THANK YOU!!!!!!!!!! 🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉
You are are hero for posting the solution! 🙇‍♂️🙇‍♂️🙇‍♂️

**Comment by petrbela** [comment]
This is up to the app's settings. This library doesn't specify a fixed version. I've updated the documentation with the recommended settings.
TL;DR If you set `castFrameworkVersion` to `21.4.0` (by default the latest version is used), you need to update `compileSdkVersion` to 34.
